### PR TITLE
Views

### PR DESCRIPTION
**Pull request by HansZizold**
### VIEWS

- The views of the Blog app were implemented according to the wireframes.
- Functionalities of the views were tested populating data in the database using the rails console.
- Links between pages were implemented.

[Complete requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md)

Happy review!!!

### CODE REVIEW

File: app/assets/stylesheets/post_index.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.comments {
```
Comment (GzimAsani) [inline app/assets/stylesheets/post_index.css:1]: - ### This inline comment is intended for README file:
- [ ] Please add on your readme file how to create  and migrate  your database
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @HansZizold                             :wave:   
 Good job so far, you are **99%!** done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your views, they work correctly  :+1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HansZizold** [comment]
@GzimAsani, thank you very much for your feedback.

I added the procedure for creating the database for your review.

Regards,

Hans

**Review by mwanawabangona — APPROVED** [review]
Hi @HansZizold 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**